airbyte-integrations/bases/base-normalization/entrypoint.sh

```diff
@@ -37,18 +37,17 @@ function configuredbt() {
   else
     # Use git repository as a base workspace folder for dbt projects
     if [[ -d git_repo ]]; then
-      echo "git_repo already exists in ${PROJECT_DIR}"
+      rm -rf git_repo
```

Comment:
We have to remove the git_repo folder in cases where multiple operations are cloning different repositories in the same sync...